sormas-api/src/main/resources/strings.properties

```diff
@@ -1284,6 +1284,7 @@ messageSormasToSormasSimilarPersonFound = There is at least one similar person i
 messageDeleteWithPendingShareRequest = There is a pending share request. With a deletion, the share request will be revoked and deleted.
 messageCannotMergeMoreThanTwoPersons = You need to select two persons for merge!
 messageAutomaticDeletionStarted = Automatic deletion has been started and will be executed in the background. Please note that, depending on the amount of data that is deleted, this process can take some time.
+messageUserUnusableForLogin = The user will be unusable for login
```

Comment:
please rename to `messageUserRoleUnusableForLogin` because it refers to user role
and update the message to *Users with only this role will not be able to login because the role does not have "Access Sormas UI" nor "Access Sormas REST" right*

Reply:
ok, I did the requested changes